src/rabbit_misc.erl

```diff
@@ -71,6 +71,7 @@
 -export([store_proc_name/1, store_proc_name/2]).
 -export([moving_average/4]).
 -export([get_env/3]).
+-export([get_channel_operation_timeout/0]).
```

Comment:
Shouldn't this function be in `rabbit_channel`, since it defines a channel setting?